content_blocker_update.sh

```diff
@@ -1,13 +1,12 @@
 #!/bin/sh
 
-#
 # This Source Code Form is subject to the terms of the Mozilla Public
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-# Update commit hash when you need to update shavar prod list
-# Note: we can update this to use a tag / branch in future
-SHAVAR_COMMIT_HASH="3910527004252af3aa9dd701566a2cb3b78e5c3a"
+
+# Version 107.0 hash
+SHAVAR_COMMIT_HASH="1d71be25893b05ba85850078b1ccedc0824ca958"
```

Comment:
@lmarceau are we still doing it via commit, I thought we would be using the versions 🤔

Reply:
Isabel will change that part in the upcoming PRs. The solution for automation is listed [here](https://docs.google.com/document/d/1kA1-dr3wAxqn4Y2-rmFoofd17YU7F1fhXI4OC7mU0F4/edit#heading=h.5x0cx9dogj48). 

In the meantime, what I am doing is using 107 file through that commit. So at least we have the new file format while automation solution is implemented.